### PR TITLE
For #60: Allow to import unnamed items (history)

### PR DESCRIPTION
**Pull request by mohierf**

### CODE REVIEW

File: alignak_backend_client/backend_client.py [hunk 1]
```diff
@@ -811,7 +811,8 @@ def create_update_resource(self, resource_name, name, update=False):
             count = 0
             for json_item in json_data:
                 logger.info("-> json item: %s", json_item)
-                if name is None and ('name' not in json_item or not json_item['name']):
+                if resource_name not in ['history'] and name is None \
```
Comment (ddurieux) [inline alignak_backend_client/backend_client.py]: You have too the ressources `userrestrictrole`, `logcheckresult`, `alignakretention`, `actionforcecheck`, `actiondowntime` and `actionacknowledge` that not have `name` field

Comment (mohierf) [reply alignak_backend_client/backend_client.py]: No need to import such resource with this utility

Comment (mohierf) [reply alignak_backend_client/backend_client.py]: Perharps userrestrictrole indeed... I will improve this

**Comment by mohierf** [comment]
@ddurieux : please merge this

**Pull request merged**